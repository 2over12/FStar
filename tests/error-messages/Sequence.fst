module Sequence

let blah () : unit = ()

let test () =
  blah();
  blah();
  123
