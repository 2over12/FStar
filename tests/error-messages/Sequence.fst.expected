Module after desugaring:
module Sequence
Declarations: [
let blah _ = () <: Prims.unit
let test _ =
  Sequence.blah ();
  Sequence.blah ();
  123
]
Exports: [
let blah _ = () <: Prims.unit
let test _ =
  Sequence.blah ();
  Sequence.blah ();
  123
]

Module before type checking:
module Sequence
Declarations: [
let blah _ = () <: Prims.unit
let test _ =
  Sequence.blah ();
  Sequence.blah ();
  123
]
Exports: [
let blah _ = () <: Prims.unit
let test _ =
  Sequence.blah ();
  Sequence.blah ();
  123
]

Module after type checking:
module Sequence
Declarations: [
let blah _ = () <: Prims.unit
let test _ =
  Sequence.blah ();
  Sequence.blah ();
  123
]
Exports: [
let blah _ = () <: Prims.unit
let test _ =
  Sequence.blah ();
  Sequence.blah ();
  123
]

Verified module: Sequence
All verification conditions discharged successfully
